src/module/actor/character/crafting/ability.ts

```diff
@@ -128,31 +128,33 @@ class CraftingAbility implements CraftingAbilityData {
             return false;
         }
 
-        if (formula.item.level > this.maxItemLevel) {
+        // Check if it meets the entry requirements
+        const rollOptions = formula.item.getRollOptions("item");
+        const viable = this.craftableItems.find((c) => c.predicate.test(rollOptions));
```

Comment:
Confused by this: what is crafting viability?

Reply:
Viable crafting definitions, but this is an artifact. viable is checked to find the best batch size, but we don't need that here. I tweaked it just now.